subsys/net/lib/lwm2m/lwm2m_shell.c

```diff
@@ -11,16 +11,37 @@ LOG_MODULE_REGISTER(LOG_MODULE_NAME);
 
 #include <stddef.h>
 #include <stdlib.h>
+#include <stdio.h>
 
 #include <zephyr/kernel.h>
 #include <zephyr/net/lwm2m.h>
 #include <zephyr/shell/shell.h>
+<<<<<<< HEAD
```

Comment:
I see you have a small issue here - I assume you tried to rebase your changes but some artifacts were left which should be removed.